miner/worker.go

```diff
@@ -46,7 +46,7 @@ const (
 	txChanSize = 4096
 
 	// chainHeadChanSize is the size of channel listening to ChainHeadEvent.
-	chainHeadChanSize = 10
+	chainHeadChanSize = 1024
```

Comment:
Can this be a DoS vector? There must have been a reason for it to be 10 initially

Reply:
Ended up reverting the value to `10`, I don't think it has much of an impact on the dos'ability of the node but it also didn't need to be changed